variables.tf

```diff
@@ -25,9 +25,21 @@ variable "dd_site" {
 }
 
 variable "integration_name" {
-  type = string
+  type        = string
   description = "Name of the integration to create."
-  default = "Datadog metrics"
+  default     = "Datadog"
+}
+
+variable "send_logs" {
```

Comment:
Just a nit: maybe call this `send_run_logs` to make it more obvious exactly what logs are being shipped?